plugins/tech-radar/src/components/RadarComponent.tsx

```diff
@@ -47,6 +48,29 @@ const useTechRadarLoader = (props: TechRadarComponentProps) => {
 const RadarComponent = (props: TechRadarComponentProps): JSX.Element => {
   const { loading, error, value: data } = useTechRadarLoader(props);
 
+  const mapToEntries = (
+    loaderResponse: TechRadarLoaderResponse | undefined,
+  ): Array<Entry> => {
+    return loaderResponse!.entries.map(entry => {
```

Comment:
This one accepts `undefined` and then immediately does `!`, which sounds risky. Did you mean to do `?.` and return the empty array as a fallback or something like that?